app/src/features/Mappings/MappingNameDialog.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useEffect, useState } from "react";
```

Comment:
I think we need to discuss on where we put the files since we're not really splitting them between features as we should. I don't think we should keep following the data model structure. For instance, the files which concern the mapping creation may be grouped together in a separate folder.

Reply:
I feel the same

Reply:
yep could be it! Also what could be a good indicator of where putting a component is what slice(s) it's using.. The only thing I would want is to not have too deep folder nesting (what you're suggesting is fine but Cohort-style Mariana trench deep folder nesting not so much)